### DISCUSSION TITLE
Want to Know How to Run Benchmark

### DISCUSSION THREAD

**Discussion by lcrLeander** [sotopia-lab/sotopia#163] [Q&A] [answered]
The instruction in https://docs.sotopia.world/examples/benchmark just doesn't work. I have tried many ways but still cannot figure out how to run the benchmark for a model (e.g. gpt-3.5-turbo, having set openAI Key).

**Comment by XuhuiZhou** [comment]
@ProKil , I believe there's an API update

**Comment by ProKil** [comment, marked as answer]
@Xiaohongdian119  Sorry for the lack of documentation for our new benchmark tool. You can do `sotopia benchmark --help` to view the usage for now. We are preparing a new version of doc to cover this.
